WORKSPACE

```diff
@@ -323,6 +323,29 @@ maven_install(
     jetify = True,
 )
 
+# https://github.com/bazelbuild/rules_jvm_external/issues/351
+maven_install(
+    name = "json_artifacts_testing",
+    artifacts = [
+        "io.quarkus:quarkus-maven-plugin:1.0.1.Final",
+        "io.quarkus:quarkus-bom-descriptor-json:1.0.1.Final",
+    ],
+    maven_install_json = "//tests/custom_maven_install:json_artifacts_testing_install.json",
+    repositories = [
+        "https://repo.maven.apache.org/maven2/",
+        "https://repo.spring.io/plugins-release/",
+    ],
+    fetch_sources = True,
+    use_unsafe_shared_cache = True,
```

Comment:
I don't think there's a need to test with fetch_sources and use_unsafe_shared_cache.

Reply:
I just removed `use_unsafe_shared_cache`